ooni/nettests/blocking/telegram.py

```diff
@@ -0,0 +1,90 @@
+# -*- encoding: utf-8 -*-
+
+from twisted.internet import defer, reactor
+from twisted.python import usage
+from twisted.internet.endpoints import TCP4ClientEndpoint
+
+from ooni.utils import log
+from ooni.common.tcp_utils import TCPConnectFactory
+from ooni.errors import failureToString
+
+from ooni import nettest
+
+# These are taken from:
+# https://github.com/telegramdesktop/tdesktop/blob/e6d94b5ee7d96a97ee5976dacb87bafd00beac1d/Telegram/SourceFiles/config.h#L205
+TELEGRAM_DCS = [
+    (1, "149.154.175.50", 443),
+    (2, "149.154.167.51", 443),
+    (3, "149.154.175.100", 443),
+    (4, "149.154.167.91", 443),
+    (5, "149.154.171.5", 443)
```

Comment:
A number of Telegram implementations use this DC:
https://github.com/NetherX/Telegram-CSharp/blob/2032ff8ecea44576e9144f6dfe1af54de91d2969/TLSharp.Core/Network/TcpTransport.cs#L11
https://github.com/LonamiWebs/Telethon/blob/af9fbbb4635515efb76c4a01deacbf7280c9dff3/telethon/tl/session.py#L13
https://github.com/miladsoft/TelegramDesktop/blob/64c14d5d69e00fdd60c9a8bf21057164a0d85323/TLSharp.Tests/TLSharpTests.cs#L189
https://github.com/badoualy/kotlogram/blob/e2c812e2c08d2d6a14ebdc9953439ddb06d0e5ff/api/src/main/kotlin/com/github/badoualy/telegram/api/Kotlogram.kt#L45